### PR TITLE
update server port change message to make it more clear

### PR DESCRIPTION
**Pull request by SummerSun**

### CODE REVIEW

**Comment by SummerSun** [comment]
#91

File: src/Tomcat/TomcatController.ts [hunk 1]
```diff
@@ -264,21 +264,17 @@ export class TomcatController {
...
-                    vscode.window.showErrorMessage(localize('tomcatExt.serverPortChangeError', `Changing the server port of a running server will cause errors, please change it back to ${ serverPort} ！`));
-                } else if (httpPort !== await Utility.getPort(serverConfig, Constants.PortKind.Http) ||
-                           httpsPort !== await Utility.getPort(serverConfig, Constants.PortKind.Https)) {
-                    const promptString: string = localize('tomcatExt.configChanged',
-                                                          'server.xml of running server {0} has been changed. Would you like to restart it?',
-                                                          serverName);
-                    const item: vscode.MessageItem = await vscode.window.showInformationMessage(promptString, DialogMessage.yes, DialogMessage.no);
+                    vscode.window.showErrorMessage(DialogMessage.getServerPortChangeErrorMessage(serverName, serverPort));
+                } else if (httpPort !== await Utility.getPort(serverConfig, Constants.PortKind.Http) || httpsPort !== await Utility.getPort(serverConfig, Constants.PortKind.Https)) {
```
Comment (ansyral) [inline src/Tomcat/TomcatController.ts]: is this a `or` relationship?

Comment (SummerSun) [reply src/Tomcat/TomcatController.ts]: check server port change first, because if server port change, we don't take further actions.

**Comment by SummerSun** [comment]
Will provide further actions to change server port back and resend a PR.

**Pull request closed without merging**